README.md

```diff
@@ -79,14 +79,18 @@ To get started, check out [our contributing guidelines](https://github.com/ME-IC
 Want to learn more about our plans for developing ``tedana``?
 Have a question, comment, or suggestion?
 Open or comment on one of [our issues](https://github.com/ME-ICA/tedana/issues)!
-
-We ask that all contributions to ``tedana`` respect our [code of conduct](https://github.com/ME-ICA/tedana/blob/master/CODE_OF_CONDUCT.md).
-
 If you're not sure where to begin, 
 feel free to pop into [Gitter](https://gitter.im/ME-ICA/tedana) and 
 introduce yourself! We will be happy to help you find somewhere to get
 started.
 
+We ask that all contributors to ``tedana`` across all project-related 
+spaces (including but not limited to GitHub, Gittter and project e-mails),
```

Comment:
Suggested change (commas, spelling, and hyphen in email):
> (including, but not limited to, GitHub, Gitter, and project emails)

Reply:
Is Oxford comma preferred on the project? I could have sworn I saw sentences without it... Anyway, I like Oxford comma so I'm not going to argue.

Reply:
I honestly didn't realize the commas were optional in this case, but it turns out [they totally are](https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Commas/faq0060.html). Sorry about that! Feel free to ignore that request.

The only thing that actually needs to be changed is Gittter --> Gitter.

Reply:
That's okay, commas are shockingly contentious and ambiguous, probably because the guideline to put commas where people pause in speech doesn't map well to some of their uses in written instruction, plus variability in how people talk. Good ol' English for you. I agree a colon would make sense though, I'll drop that in. 

I cannot believe I keep missing the three t's, apparently I'm being blind this morning, maybe everything I type should be regarded with extra suspicion. I partially blame my MacBook's sad little chiclet keyboard.

Reply:
Oh okay, I feel better; it turns out that GitHub is still rendering the out-of-date version; if you look at the latest commit Gitter is spelled correctly.